resources/view/product/edit-images.html.twig

```diff
@@ -6,9 +6,22 @@
 			<h2>{{ type }}</h2>
 			{% for image in imgs %}
 			<div class="product-image">
+				<div class="close">
+					{{ render(controller('Message:Mothership:Commerce::Controller:Product:Edit#deleteImageForm', {imageID: image.id})) }}
+				</div>
 				{{ getResizedImage(image, 200, 200) }}
 				<div class="file-overlay">
 					{{ image.file.name }}
+					<dl>
+						<dt>Image Type</dt>
+						<dd>{{ image.type }}</dd>
+						{% if image.options %}
+						{% for key, value in image.options %}
+							<dt>{{ key }}</dt>
```

Comment:
Possibly. Though it may make it look better, I don't like it. In mothership 'Colour' is different to 'colour' so it's useful to be able to see what it actually is.